packages/editor/src/panels/assets/topbar.tsx

```diff
@@ -77,9 +78,11 @@ const ViewModeSettings = () => {
 
 const uploadFiles = () => {
   const projectName = getState(EditorState).projectName
+  const importFolder = getState(ImportSettingsState).importFolder || '/assets/'
+
   return inputFileWithAddToScene({
     projectName: projectName as string,
-    directoryPath: `projects/${projectName}/assets/`
+    directoryPath: `projects/${projectName}${importFolder}`
```

Comment:
This will allow importing to any folder inside the project. We should allow uploading only inside `public` and `assets` folders.
@HexaField what are your thoughts?

Reply:
Yeah we need to limit to just these folders

Reply:
fixed. We're only allowed to upload to public and assets folder. We also needed to check whether the path ends with '/'